styleguide/source/_patterns/00-base/intro/intro.twig

```diff
@@ -1,35 +0,0 @@
-<div class="sg-introduction container">
-  <h2>{% include 'site-logo.twig' %}</h2>
-  <h1>Welcome to the AMA Style Guide</h1>
-  <p>Click the "all" link or navigation above to browse styles.
-    You can search for patterns using "Find Pattern" in the gear on the right.
-    View the {% include 'link.twig' with {'content' : 'latest release' , 'url' : 'https://github.com/AmericanMedicalAssociation/AMA-style-guide/releases/latest', 'target': '_blank', 'class': 'ama__link-blue'} %}
-
-    <img src="https://img.shields.io/github/release/AmericanMedicalAssociation/AMA-style-guide.svg" alt="Latest Release"></a> or
-    {% include 'link.twig' with {'content' : 'development code', 'url' : 'https://github.com/AmericanMedicalAssociation/AMA-style-guide'}%}
-    on GitHub.
-  </p>
-  <h2>Working with our style guide</h2>
-  <h3>Structure</h3>
-  <p>This living style guide is built with
-    {% include 'link.twig' with {'content' : 'Pattern Lab' , 'url' : 'http://patternlab.io/', 'target': '_blank', 'class': 'ama__link-blue'} %}
-    and adheres to the principles of
-    {% include 'link.twig' with {'content' : 'Atomic Design' , 'url' : 'http://bradfrost.com/blog/post/atomic-web-design/', 'target': '_blank', 'class': 'ama__link-blue'} %}.
-  </p>
-  <h3>Issues</h3>
-  <p>If you have questions or encounter an issue working with the style guide, report it as
-  {% include 'link.twig' with {'content' : 'a GitHub issue', 'url' : 'https://github.com/AmericanMedicalAssociation/AMA-style-guide/issues', 'class': 'ama__link-blue'} %}.
-  </p>
-  <h3>Contributions</h3>
-  <p>For any code that you would like to contribute to the style guide, first create an issue in Github explaining clearly what problem you are trying to solve.</p>
-  <p>Make sure you have adhered to the guidelines outlined in the
-  {% include 'link.twig' with {'content' : 'Code Conventions documentation' , 'url' : 'https://github.com/AmericanMedicalAssociation/AMA-style-guide/blob/develop/docs/code_conventions.md', 'class': 'ama__link-blue'} %}
-    before submitting a pull request. Reference the Github issue you created when you submit your pull request.
-  </p>
-  <h3>Annotation and Documentation</h3>
-  <p>Each pattern should have its own accompanying
-  {% include 'link.twig' with {'content' : 'Markdown' , 'url' : 'https://daringfireball.net/projects/markdown/', 'target': '_blank', 'class': 'ama__link-blue'} %}
-  file that contains 1) a useful description of the pattern and 2) link to the relevant Jira tickets/Github issues relating to that pattern's creation.
-  This information is needed to provide context as to how and when  a particular pattern should be used.
-  </p>
-</div>
```

Comment:
This file was deleted because it was in the wrong place. It needs to be in pages not in base.

Reply:
Idk if pages is the right place for it either; how does Mayflower do this?

Reply:
In pages